srml/contract/src/lib.rs

```diff
@@ -17,11 +17,11 @@
 //! # Contract Module
 //!
 //! The contract module provides functionality for the runtime to deploy and execute WebAssembly smart-contracts.
-//! The supported dispatchable functions are documented as part of the [`Call`](./enum.Call.html) enum.
+//! To use it in your module, you need to implement the contracts [`Trait`](./trait.Trait.html).
 //!
 //! ## Overview
 //!
-//! This module extends accounts (see `Balances` module) to have smart-contract functionality.
+//! This module extends accounts (see the balances module) to have smart-contract functionality.
```

Comment:
Not true. It just relies on currency so it can be used with any module that implements that.